test/Microsoft.Azure.Devices.Edge.Test.Common/linux/PackageManagement.cs

```diff
@@ -105,10 +134,11 @@ public string[] GetInstallCommandsFromMicrosoftProd(Option<Uri> proxy)
             },
             SupportedPackageExtension.Rpm => new[]
             {
-                "yum remove -y --remove-leaves aziot-edge",
-                "yum remove -y --remove-leaves aziot-identity-service",
-                "yum remove -y --remove-leaves iotedge",
-                "yum remove -y --remove-leaves libiothsm-std",
+                "yum remove -y aziot-edge",
+                "yum remove -y aziot-identity-service",
```

Comment:
What about `aziot-certd`, `aziot-identityd`, etc? Are those automatically removed with the uninstallation of the `aziot-edge`?

Reply:
The `--remove-leaves` option was taking take of the dependent packages before this changes, but in Red Hat 8, "yum" is an alias to "dnf" and this option didn't work.  I should achieve the same by invoking `yum autoremove -y`.